KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/Utils.java

```diff
@@ -231,7 +231,14 @@ public static String getStreamName(IMediaStream stream) {
     }
 
     public static String getMediaServerHostname() throws IOException, InterruptedException {
-        Process p = Runtime.getRuntime().exec("hostname -f");
+        return getMediaServerHostname(true);
+    }
+
+    public static String getMediaServerHostname(boolean full) throws IOException, InterruptedException {
+        String command = "hostname";
+        if (full)
```

Comment:
Disregard I now saw this is Anne's function you left the one that and sent true.